style.css

```diff
@@ -0,0 +1,386 @@
+	border-bottom: 50px solid  #355C7D ;
+	background-color: #a1596e;
+
+}
+
+.portfolio-title-container {
+	max-width: 1000px;
+	width: 100%;
+	padding: 15px;
+}
+
+.projects-container {
+	width: 100%;
+	max-width: 1000px;
+	display: flex;
+	flex-direction:column;
+	justify-content: flex-start;
+	flex-wrap: wrap;
+	gap: 40px;
+	padding: 15px;
+}
+
+.project {
+	display: flex;
+	justify-content: space-around;
+	flex-direction: column; 
+	gap: 10px;
+	border: 5px solid #C06C84;
+	border-radius: 2px;
+	background-color: #355C7D;;
+
+	
+}
+.project > h3 {
+	font-size: 2.5rem;
+	margin: 25px;
+	margin-bottom: 0;
+	letter-spacing: 2px;
+}
+
+.project-content{
+	display: flex;
+	flex-direction: row;
+	width: 100%;
+	max-width: 1000px;
+	justify-content: space-between;
+	flex-wrap: no-wrap;
+	
+}
+
+.project-content img {
+	width: 350px;
+	height: 200px;
+	padding: 10px;
+
+}
+
+
+
+.project-words {
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	
+}
+
+.project-words > p {
+	font-size: 2rem;
+	padding: 10px;
+	letter-spacing: 3px;
+	line-spacing: 40px;
+}
+
+.project-links-container {
+	display:flex;
+	flex-direction: row;
+	justify-content: flex-end;
+	gap: 20px;
+	margin-right: 30px;
+}
+
+.project-links-container > a {
+	font-size: 1.5rem;
+}
+
+.contact {
+	background-color: #355C7D;
+}
+.contact-outer-container {
+	display: flex;
+	justify-content: flex-start;
+	align-items: flex-start;
+	flex-direction: column;
+	max-width: 1000px ;
+	margin: 0 auto;
+	background-color: #355C7D;
+	padding: 15px;
+}
+
+.contact-outer-container > h2 {
+	font-size: 3rem;
+	margin: 15px;
+	letter-spacing: 3px;
+}
+
+.contact-outer-container > p {
+	font-size: 2.5rem;
+	margin: 15px;
+	letter-spacing: 3px;
+	line-height: 30px;
+}
+
+.contact-container {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	width: 100%;
+}
+
+.contact-container > a> img {
+	max-width: 100px;
+	margin:15px;
+}
+.contact-top {
+	display:none;
+}
+
+footer a {
+	font-size: 1.5rem;
+}
+
+footer p {
+	margin-top: 15px;
+}
+
+
+@media screen and (max-width:800px) {  
+
+	nav{
+		display: none;	
+	}
+	
+	.intro-texts{
+		display: flex;
+		flex-direction: column;
+		justify-content: center;
+		align-items: center;
+		text-align: center;
+	}
+	.intro-texts > h1 {
+		font-size: 5rem;
+		line-height: 50px;
+	}
+	.intro-texts > h3 {
+		font-size: 2.4rem;
+		letter-spacing: 2px;
+		line-height: 35px;
+	}
+	
+	br {
+		display: inline;
+	}
+	
+	.intro {
+		gap: 0px;
+	}
+	.intro > img {
+		display: none;
+	}
+	.contact-top {
+		display: block;
+		max-height: 80px;
+		position: fixed;
+		width: 100%;
+	}
+	.contact-bottom {
+		display: none;
+	}
+	
+	.about-me-text{
+		max-width:100%;
+	}
+	
+	.about-me > img {
+		width:100%;
+	}
+	
+	.about-me-text > p, .portfolio-title-container > h3{
+		letter-spacing: 1.5px;
+		line-height: 22px;
+
+	}
+	
+	.contact-container {
+		justify-content: center;
+		gap: 90px;
+	}
+	.contact-container > a> img {
```

Comment:
really this should be a class called `contact-icon`

Reply:
Changed!